File: packages/scanner/tests/fixtures/VCS-014/clean.py
import sqlite3

def get_user(user_id):
    conn = sqlite3.connect("app.db")
    cursor = conn.cursor()
    cursor.execute("SELECT * FROM users WHERE id = ?", (user_id,))
    return cursor.fetchone()

def update_status(uid, status):
    cursor.execute("UPDATE users SET status = ? WHERE id = ?", (status, uid))

# Plain string queries are fine if there are no variables
def get_all():
    cursor.execute("SELECT * FROM users")

File: packages/scanner/tests/fixtures/VCS-014/clean.js
async function getUser(id) {
  return await db.query("SELECT * FROM users WHERE id = ?", [id]);
}

async function search(name) {
  return await db.query("SELECT * FROM products WHERE name = $1", [name]);
}

async function getAll() {
  return await db.query("SELECT * FROM users");
}
